.github/workflows/acceptance.yml

```diff
@@ -0,0 +1,199 @@
+name: acceptance
+on: pull_request
+
+jobs:
+    percynonce:
+      name: Generate Percy nonce
+      runs-on: ubuntu-latest
+      steps:
+        - name: Generate Percy nonce
+          if: always()
+          run: |
+            echo ${{ github.run_id }}.$(($(date +%s))) > nonce.txt
+
+        - name: Upload nonce as artifact
+          if: always()
+          uses: actions/upload-artifact@v1
+          with:
+            name: percy
+            path: nonce.txt
+
+    acceptance:
+      needs: [percynonce]
+      runs-on: ubuntu-16.04
+      strategy:
+        matrix:
+          instance: [0, 1, 2]
+
+      env:
+        PIP_DISABLE_PIP_VERSION_CHECK: on
+        PIP_QUIET: 1
+
+        SENTRY_LIGHT_BUILD: 1
+        SENTRY_SKIP_BACKEND_VALIDATION: 1
+        MIGRATIONS_TEST_MIGRATE: 0
+
+        # Use this to override the django version in the requirements file.
+        DJANGO_VERSION: ">=1.11,<1.12"
+
+        # Node configuration
+        # node's version is pinned by .nvmrc and is autodetected by `nvm install`.
+        NODE_OPTIONS: --max-old-space-size=4096
+        NODE_ENV: development
+
+        PYTEST_SENTRY_DSN: https://6fd5cfea2d4d46b182ad214ac7810508@sentry.io/2423079
+        PYTEST_ADDOPTS: "--reruns 5"
+
+        # services configuration
+        SENTRY_KAFKA_HOSTS: kafka:9093
+        SENTRY_ZOOKEEPER_HOSTS: zookeeper:2182
+        SENTRY_REDIS_HOST: redis
+        # The hostname used to communicate with the PostgreSQL service container
+        POSTGRES_HOST: postgres
+        # The default PostgreSQL port
+        POSTGRES_PORT: 5432
+
+        # Percy config
+        PERCY_TOKEN: ${{ secrets.PERCY_TOKEN }}
+        PERCY_PROJECT: ${{ secrets.PERCY_PROJECT }}
+
+        # Number of matrix instances
+        TOTAL_TEST_GROUPS: 3
```

Comment:
Whoever added `TOTAL_TEST_GROUPS` to our original builds, that is some GOOD STUFF.

Reply:
Agreed, I believe it was @wedamija

Reply:
I vaguely remember seeing dan on git blame for that few months back when I was curious, good stuff Dan.

Reply:
Spring cleaning was a good week, we made lots of CI improvements at the time. Looking forward to being able to use this parallelization properly and get acceptance tests below 20 minutes.

Reply:
@wedamija  I commented on this, not sure if you saw: https://github.com/getsentry/sentry/pull/18576#issuecomment-623668920